src/puppetlabs/puppetdb/http/command.clj

```diff
@@ -21,21 +21,30 @@
 
 (def valid-commands-str (str/join ", " (sort (vals command-names))))
 
+(defn coerce-to-int
+  [s]
+  (if (number? s) s (Integer/parseInt s)))
```

Comment:
It looks like below you're using `Integer.` for this, could you the same thing be done in both places?